SwiftReflector/OverrideBuilder.cs

```diff
@@ -837,7 +840,11 @@ SLProperty DefineOverrideProperty (FunctionDeclaration getter, FunctionDeclarati
 				var ifelse = new SLIfElse (condition, ifblock, elseblock);
 
 				if (isProtocol) {
-					getBlock = ifblock;
+					if (getBlock.Count == 0)
+						getBlock = ifblock;
+					else {
```

Comment:
Nitpick: if one part of the if statement has curly braces, the other should have too (or neither).

Reply:
Good to know. Likely this test is going to go away in about 4 PRs so I'm not going to fix it now.